Possible build error

Test server : 
* Ubuntu 14.04.2 LTS fresh install
* go version go1.3 linux/amd64 installed from sources
* lxd commit 4b84c8bd047ae1b71352b76677fd60d3b21b981a
* installed packages : 
```
acl xzip tar rsync pxz xz-utils
python-lzma cgroup-bin libpam-systemd

lxc lxc-dev mercurial git pkg-config
protobuf-compiler golang-goprotobuf-dev
```
* ```protoc``` & ```protoc-gen-go``` in path (from ubuntu official repos)

```bash
> protoc --version
libprotoc 2.5.0

> export GOBIN=/opt/go/bin
> export GOPATH=/opt/go
> export PATH=$PATH:/usr/local/go/bin
> cd /opt/go/src/github.com/lxc/lxd

> /usr/local/go/bin/go get -v -u -d ./...
github.com/lxc/lxd (download)
github.com/gorilla/websocket (download)
github.com/gosexy/gettext (download)
github.com/mattn/go-sqlite3 (download)
gopkg.in/lxc/go-lxc.v2 (download)
golang.org/x/crypto (download)
gopkg.in/yaml.v2 (download)
github.com/olekukonko/tablewriter (download)
code.google.com/p/go-uuid (download)
github.com/dustinkirkland/golang-petname (download)
github.com/golang/protobuf (download)
github.com/gorilla/mux (download)
github.com/gorilla/context (download)
Fetching https://gopkg.in/tomb.v2?go-get=1
Parsing meta tags from https://gopkg.in/tomb.v2?go-get=1 (status code 200)
get "gopkg.in/tomb.v2": found meta tag main.metaImport{Prefix:"gopkg.in/tomb.v2", VCS:"git", RepoRoot:"https://gopkg.in/tomb.v2"} at https://gopkg.in/tomb.v2?go-get=1
gopkg.in/tomb.v2 (download)
```
The error happens when i'm trying to run make : 
```bash
> make
protoc --go_out=. ./lxd/migration/migrate.proto
go install -v ./...
lxd/migration/migrate.pb.go:7:8: cannot find package "code.google.com/p/goprotobuf/proto" in any of:
	/usr/local/go/src/pkg/code.google.com/p/goprotobuf/proto (from $GOROOT)
	/opt/go/src/code.google.com/p/goprotobuf/proto (from $GOPATH)
make: *** [default] Error 1
```

Is there another way to build protoc ? (I tried to rebuild protoc from sources but it did the same) ... And protoc doesn't seem to be in go get dependencies list (normal ?)
Is it normal at this point ? Or did I made a mistake somewhere ?

--- a/lxd/migration/migrate.go
+++ b/lxd/migration/migrate.go
@@ -12,7 +12,17 @@ import (
 	"net/url"
 	"os"
 
-	"github.com/golang/protobuf/proto"
+	/*
+	 * Although the goprotobuf project has moved to github, the protoc
+	 * compiler still generates this import. Since this the only file in
+	 * the tree (i.e. the migrate.pb.go file is generated during make),
+	 * when someone does `go get -u ./...`, they don't have any
+	 * dependencies that the generated protobuf file has, namely, this
+	 * dependency. Presumably protoc will switch at some point to
+	 * generating a github.com import, and then we can switch this back
+	 * too.
+	 */
+	"code.google.com/p/goprotobuf/proto"
 	"github.com/gorilla/websocket"
 	"github.com/lxc/lxd"
 	"github.com/lxc/lxd/shared"

